actions.yaml

```diff
@@ -8,3 +8,10 @@ add-admin-user:
       type: string
       description: User password.
   required: [email, password]
+anonymize-user:
+  description: Anonymize a user.
+  params:
+    username:
+      type: string
+      description: Username.
```

Comment:
does it make sense to add a redundant description? (it is equal to the name of the field)

Reply:
Any suggestion?

Reply:
Maybe `The unique identifier of the user to anonymize`?